### PR TITLE
1471 october annual function changes

### PR DESCRIPTION
**Pull request by IlseFerris**
October mass changes to functions. See commits for source of information for confirming amounts.

### CODE REVIEW

File: MASTER FUNCTIONS LIBRARY.vbs [hunk 1]
```diff
@@ -5885,7 +5885,18 @@ function determine_130_percent_of_FPG(footer_month, footer_year, hh_size, fpg_13
...
+			If hh_size = 1 Then fpg_130_percent = 1580
+			If hh_size = 2 Then fpg_130_percent = 2137
+			If hh_size = 3 Then fpg_130_percent = 2694
+			If hh_size = 4 Then fpg_130_percent = 3250
+			If hh_size = 5 Then fpg_130_percent = 3807
+			If hh_size = 6 Then fpg_130_percent = 4364
+			If hh_size = 7 Then fpg_130_percent = 4921
+			If hh_size = 8 Then fpg_130_percent = 5478
+
+			If hh_size > 8 Then fpg_130_percent = 5052 + (557 * (hh_size-8))
```
Comment (MDR22) [inline MASTER FUNCTIONS LIBRARY.vbs:5898]: Typo - should be 5478 + (557 * (hh_size-8)) to reflect 2023 amount.

**Review by MDR22 — CHANGES_REQUESTED** [review]
One small typo on HH size over 8 for 2023. Otherwise, looks good.

**Review by C-Love — CHANGES_REQUESTED** [review]
I agree with @MDR22 review but approve everything else

File: MASTER FUNCTIONS LIBRARY.vbs (hunk 1 above)
Comment (IlseFerris) [reply MASTER FUNCTIONS LIBRARY.vbs]: Good catch @MDR22, fixed!

**Pull request merged**